db/migrate/20150825121144_create_monthly_reports.rb

```diff
@@ -1,7 +1,7 @@
 class CreateMonthlyReports < ActiveRecord::Migration
   def change
     create_table :monthly_reports do |t|
-      t.integer :users_id,          null: false
+      t.integer :user_id,          null: false
```

Comment:
__[reek]__
* [FeatureEnvy] refers to t more than self (maybe move it to another class?)